src/site/content/en/lighthouse-accessibility/aria-required-parent/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Ensure child roles are contained within required parent roles.
+description: |
+  Learn about aria-required-parent audit.
+author: megginkearney
+web_lighthouse:
+  - aria-required-parent
+---
+
+[ARIA](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)
+roles and attributes help screen readers
+provide missing information about an element.
+For these roles and attributles to make sense,
+each ARIA `role` supports a specific subset of `aria-*` attributes
+(see [ARIA roles definitions](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)).
+Some ARIA child roles must be contained by specific parent roles
+to properly perform their intended accessibility functions.
+Lighthouse reports missing parenrts for these child roles:
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="aria-required-parent.png" alt="Lighthouse: ARIA role missing required parent role">
```

Comment:
I think this image is for a different audit.

Reply:
I've got a new screen shot.